### PR TITLE
feat: added `CreateMany`

### PR DESCRIPTION
**Pull request by mattia-fumo**

### CODE REVIEW

File: client_test.go [hunk 1]
```diff
@@ -900,6 +900,89 @@ func TestCreate(t *testing.T) {
...
+				Field: "something2",
+			},
+		},
+	}
+
+	expectedBody, err := json.Marshal(resourcesToCreate)
+	require.NoError(t, err)
+
+	t.Run("creates resource", func(t *testing.T) {
+		testhelper.NewGockScope(t, baseURL, http.MethodPost, "").
```
Comment (davidebianchi) [inline client_test.go:933]: Here should be "bulk"? Also in other tests. Why do not break the test?

**Review by davidebianchi — CHANGES_REQUESTED** [review]
Changelog is missing

File: client_test.go (hunk 1 above)
Comment (mattia-fumo) [reply client_test.go:933]: I suspect its still matching the prefix :thinking:

Comment (mattia-fumo) [reply client_test.go:933]: Turns out it passes because `/resource-path/bulk` matches `$`

Comment (mattia-fumo) [reply client_test.go]: I think we should enforce a full path match in https://github.com/mia-platform/go-crud-service-client/blob/main/testhelper/gock.go#L98 with `^%s$`.

Comment (davidebianchi) [reply client_test.go]: Yes, it is probably the best. We should address it. Can you open an issue?

Comment (mattia-fumo) [reply client_test.go]: https://github.com/mia-platform/go-crud-service-client/issues/28

**Comment by mattia-fumo** [comment]
> Changelog is missing

@davidebianchi couldn't find one. is it automated maybe?

**Pull request merged**